include/follow.php

```diff
@@ -102,62 +102,53 @@ function new_contact($uid,$url,$interactive = false) {
 		if ($interactive) {
 			if (strlen($a->path)) {
 				$myaddr = bin2hex(App::get_baseurl() . '/profile/' . $a->user['nickname']);
-			}
-			else {
+			} else {
 				$myaddr = bin2hex($a->user['nickname'] . '@' . $a->get_hostname());
 			}
 
 			goaway($ret['request'] . "&addr=$myaddr");
 
 			// NOTREACHED
 		}
-	}
-	else {
-		if(get_config('system','dfrn_only')) {
+	} else {
+		if (get_config('system','dfrn_only')) {
```

Comment:
Usually, the more fixes the better although in your huge PRs it may be a liability! 😅

When checking for conditional merges, please check that there are no instructions between the two `if` and/or ˋelse` statements, but also between the corresponding closing braces.